src/components/OrgPostCard/components/PostDetailModal.tsx

```
@@ -0,0 +1,279 @@
+            <Close fontSize="small" />
+          </Button>
+        </div>
+      </Modal.Header>
+
+      <Modal.Body>
+        <div className="row">
+          <div className="col-md-6 mb-3 mb-md-0">
+            {videoAttachment ? (
+              <div
+                style={{ position: 'relative' }}
+                role="region"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)